app/src/main/java/com/github/h3lp3rs/h3lp/signin/GoogleSignInAdapter.kt

```diff
@@ -19,35 +19,26 @@ import java.util.*
  * This object adapts the Google sign in and Firebase authentication to our general SignInInterface.
  * It is used as the central sign in interface in the app
  */
-object GoogleSignInAdapter: SignInInterface<AuthResult> {
+object GoogleSignInAdapter : SignInInterface<AuthResult> {
     var auth: FirebaseAuth = Firebase.auth
     lateinit var gso: GoogleSignInOptions
 
     private const val SERVER_CLIENT_ID =
-    "899579782202-t3orsbp6aov3i91c99r72kc854og8jad.apps.googleusercontent.com"
+        "899579782202-t3orsbp6aov3i91c99r72kc854og8jad.apps.googleusercontent.com"
```

Comment:
we could then put this url int the url value file too.

Then somebody wanting to use our project with other credential can easily replace our url by his